src/figuresClasses.ts

```diff
@@ -1,11 +1,95 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    this.shape = 'triangle';
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('All sides must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides cannot form a triangle');
```

Comment:
[CODE STYLE] - The error message 'Sides cannot form a triangle' could be more descriptive by explaining the triangle inequality theorem or by specifying which sides are causing the issue.